.github/dependabot.yml

```diff
@@ -6,13 +6,21 @@ updates:
     interval: daily
     time: "03:00"
   open-pull-requests-limit: 10
-  assignees:
-  - anayden
-  - artemlops
-  - YevheniiSemendiak
+  reviewers:
+    - neuro-inc/mlops
 - package-ecosystem: github-actions
   directory: "/"
   schedule:
     interval: daily
     time: "03:00"
+  reviewers:
+    - neuro-inc/mlops
   open-pull-requests-limit: 10
+- package-ecosystem: pip
+  directory: "/"
+  schedule:
+    interval: daily
```

Comment:
Do we need this daily? Maybe weekly?

Reply:
Personally, I don't see any issues with this, tbh.
We don't pay for it, neither run jobs, afaik.

On the other hand, imagine `neuro-cli` release, and on the next day, we get just a bunch of PRs, all of which will be merged within the same context (if there was a problematic CLI release - we all will keep in mind that this should not be merged yet).

Reply:
Ok, that makes sense